### DISCUSSION TITLE
I upgraded my version of flutter to (2.8.1)with the following error

### DISCUSSION THREAD

**Discussion by Miaoyu0914** [daohoangson/flutter_widget_from_html#699] [Q&A]
I upgraded my version of flutter to (2.8.1)with the following error：
Error: The method 'FwfhTextStyle.apply' has fewer named arguments than those of overridden method 'TextStyle.apply'
![error](https://user-images.githubusercontent.com/16283237/147546454-9ae9f43e-2455-47f2-9946-aa488837b368.jpg)
.

**Comment by daohoangson** [comment]
Running `flutter pub upgrade fwfh_text_style` should fix this.

**Reply by Miaoyu0914** [reply]
thanks!

**Reply by daohoangson** [reply]
Please mark as answer if it works. Thank you.
